src/os_install.c

```diff
@@ -87,15 +87,18 @@ static bool parse_opt(int opt, char *optarg)
 	case 'B': {
 		char *arg_copy = strdup_or_die(optarg);
 		char *token = strtok(arg_copy, ",");
+		bool oscore_requested = false;
 		while (token) {
-			if (strcmp(token, "os-core") != 0) {
+			if (strcmp(token, "os-core") == 0) {
```

Comment:
Why we just don't add os-core to the list?

Reply:
To be honest, IDK 🤣 , I assumed we had a reason no to, so I kept the same code logic. But now that I'm looking at the code I cannot find a reason why not to. I'll take a closer look and will run a few tests, if everything looks good I'll just add it as you mentioned.